logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/UserConfigRepository.kt

```diff
@@ -18,6 +18,8 @@ interface UserConfigRepository {
     fun setMLSEnabled(enabled: Boolean): Either<StorageFailure, Unit>
     fun setConferenceCallingEnabled(enabled: Boolean): Either<StorageFailure, Unit>
     fun isConferenceCallingEnabled(): Either<StorageFailure, Boolean>
+    fun isReadReceiptsEnabled(): Flow<Either<StorageFailure, Boolean>>
+    fun setReadReceiptsEnabled(enabled: Boolean): Either<StorageFailure, Unit>
```

Comment:
`setReadReceiptsEnabled` can be misunderstood as it will set the value to true, maybe we can rename it to `setReadReceiptsStatus`.